### ISSUE TITLE
ALB Ingress image tag 1.0-beta5 is not available

### ISSUE THREAD

**Issue by akshayks** [kubernetes-sigs/aws-alb-ingress-controller#493]
When I tried installing the latest version of the helm chart for the ALB Ingress controller, pod creation fails with `ErrImagePull`. Here is the output of `kubectl describe` on the pod:
```
Events:
  Type     Reason                 Age                From                                                 Message
  ----     ------                 ----               ----                                                 -------
  Normal   Scheduled              39s                default-scheduler                                    Successfully assigned alb-alb-ingress-controller-controller-58ff89f7b6-c2v2x to ip-10-77-88-102.us-west-2.compute.internal
  Normal   SuccessfulMountVolume  39s                kubelet, ip-10-77-88-102.us-west-2.compute.internal  MountVolume.SetUp succeeded for volume "alb-alb-ingress-controller-token-5p74m"
  Normal   Pulling                25s (x2 over 39s)  kubelet, ip-10-77-88-102.us-west-2.compute.internal  pulling image "quay.io/coreos/alb-ingress-controller:1.0-beta.5"
  Warning  Failed                 24s (x2 over 38s)  kubelet, ip-10-77-88-102.us-west-2.compute.internal  Failed to pull image "quay.io/coreos/alb-ingress-controller:1.0-beta.5": rpc error: code = Unknown desc = Error response from daemon: manifest for quay.io/coreos/alb-ingress-controller:1.0-beta.5 not found
  Warning  Failed                 24s (x2 over 38s)  kubelet, ip-10-77-88-102.us-west-2.compute.internal  Error: ErrImagePull
  Normal   BackOff                12s (x2 over 37s)  kubelet, ip-10-77-88-102.us-west-2.compute.internal  Back-off pulling image "quay.io/coreos/alb-ingress-controller:1.0-beta.5"
  Warning  Failed                 12s (x2 over 37s)  kubelet, ip-10-77-88-102.us-west-2.compute.internal  Error: ImagePullBackOff
```
It appears there is no image with the tag `1.0-beta.5`. I have worked around this issue for the time being by overriding the image tag to `latest` in a custom values.yaml. Am I doing something strange or was the image not tagged correctly?

Akshay

**Comment by bzon** [comment]
Getting the same issue.

**Comment by iggy** [comment]
https://github.com/kubernetes-sigs/aws-alb-ingress-controller/commit/5151369009a630d3e024c8b1a501a7b0d952be97#diff-773533eaa031e17f129a8bc8ea77243fR20

Merged in #476 

Looks like something snuck in during a big merge

**Comment by bigkraig** [comment]
Just pushed it up
